model/cenik.php

```diff
@@ -77,6 +78,17 @@ public function __construct(
         }
     }
 
+    public function getTextySlev(): array
```

Comment:
Pokud má být tohle jediný způsob jak přistupovat k poli $textySlev (protože přímý přístup nevrátí správnou hodnotu), tak to neměj jako pole třídy, ale proměnnou v této funkci.